kotlin/workflow-ui-android/src/main/java/com/squareup/workflow/ui/LayoutRunner.kt

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright 2019 Square Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.squareup.workflow.ui
+
+import android.content.Context
+import android.support.annotation.LayoutRes
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import kotlin.reflect.KClass
+
+/**
+ * An object that handles [View.showValue] calls for a view inflated
+ * from a layout resource in response to [ViewRegistry.buildView].
+ * (Use [BuilderBinding] if you want to build views from code rather than
+ * layouts.)
+ *
+ * Typical usage is to have a [LayoutRunner]'s `companion object` implement
```

Comment:
Can't catch that at compile time, just like today. Very strict, descriptive exception handling is in place.

Reply:
Maybe answering the wrong question. This will not compile w/o an implementation of `ViewBinding`, whether via delegating to a `bind` call or some other way.

```
val TicTacToeViewBuilders = ViewRegistry(
    NewGameLayoutRunner, GamePlayLayoutRunner, GameOverLayoutRunner
)
```